spec/models/card_spec.rb

```diff
@@ -4,14 +4,14 @@
   it 'create card with empty original text' do
     card = Card.create(original_text: '', translated_text: 'house', user_id: 1,
                        block_id: 1)
-    expect(card.errors[:original_text]).to include('Необходимо заполнить поле.')
+    expect(card.errors[:original_text]).to include('Заполните поле.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.